corehq/apps/reports/templates/reports/notification_unsubscribe.html

```diff
@@ -1,18 +1,29 @@
-{% extends "reports_core/base_template_new.html" %}
+{% extends 'style/bootstrap3/base.html' %}
```

Comment:
Need to remove `bootstrap3` now that this is merged https://github.com/dimagi/commcare-hq/pull/11881